person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
+  def initialize(_name, age, _parent_permission)
+    @id = Random.rand(1..1000)
+    @name = 'Unknown'
+    @age = age
+    @parent_permission = true
+  end
+
+  def of_age?
+    age >= 18 || parent_permission
+  end
```

Comment:
- [ ] Hi as mentioned in the requirements, please make your `of_age` method a private method. Also, this method is supposed to return true if the age is above 18 so there is no need to check parent permission in this method.